main.cpp

```diff
@@ -20,12 +20,24 @@
 #if TARGET_UBLOX_EVK_ODIN_W2
 #include "OdinWiFiInterface.h"
 OdinWiFiInterface wifi;
+
+#elif TARGET_NUCLEO_L476RG
+#include "ESP8266Interface.h"
+// ESP8266_WIFI TX to be connected to D2 (Serial1_Rx)
+// ESP8266_WIFI_RX to be connected to D8 (Serial1_Tx)
+ESP8266Interface wifi(D8, D2);
+
+#elif  TARGET_NUCLEO_F429ZI
```

Comment:
Could we not have special case for this board if the used pins are the same as for 'default' esp, it should work OOB if you use D1/D0.